### PR TITLE
Roller commands

### PR DESCRIPTION
**Pull request by YoniKiriaty**
The commands for the roller after merge

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollInForGivenTime.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class RollInForGivenTime extends ParallelRaceGroup {
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RollInForGivenTime.java:6]:
```suggestion
public class RollInForAGivenTime extends ParallelRaceGroup {
```

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRollerUntilNoteInside.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+public class RunRollerUntilNoteInside extends RollerCommand{
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRollerUntilNoteInside.java:3]:
```suggestion
public class RunRollerUntilNoteIsInside extends RollerCommand{
```

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRollerUntilObjectOut.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+public class RunRollerUntilObjectOut extends RollerCommand{
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRollerUntilObjectOut.java:3]:
```suggestion
public class RunRollerUntilObjectIsOut extends RollerCommand{
```

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/RollerConstants.java [hunk 4]
```diff
@@ -4,6 +4,7 @@ public class RollerConstants {
 
 	public static final double ROLL_FORWARD_POWER = 1;
 	public static final double ROLL_BACKWARD_POWER = -1;
+	public static final double SAFETY_TIME_DIFFERENCE_AFTER_NOTE_RELEASED = 0.5;
```
Comment (aaa-png) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/RollerConstants.java:7]:
```suggestion
	public static final double SAFETY_TIME_AFTER_NOTE_IS_RELEASED = 0.5;
```

**Review by aaa-png — CHANGES_REQUESTED** [review]

**Review by aaa-png — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/arm/roller/ReleaseNote.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.roller.RollerConstants;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class ReleaseNote extends SequentialCommandGroup {
+    public ReleaseNote(){
+        super(
+                new RunRollerUntilObjectIsOut(),
+                new RollInForAGivenTime(RollerConstants.SAFETY_TIME_AFTER_NOTE_IS_RELEASED)
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/commands/arm/roller/ReleaseNote.java:10]: roll in when trying to release note?

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 6]
```diff
@@ -59,13 +61,8 @@ public SmartJoystick getSecondJoystick() {
     }
 
     public void initButtons() {
-        secondJoystick.START.whileTrue(new PanicMode());
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/OI.java:62]: do not delete buttons

**Review by yonnbro — APPROVED** [review]

**Pull request merged**